cypress-io/cypress
# chore: valid config files should be collapsed during onboarding

- Closes #23260

### User facing changelog
- During Cypress onboarding, when config files are created, all valid, skipped, error config files will remain collapsed on default while warning files will be open on default.

### Additional details

- This reduces the information shown to the user on a 'config success' screen making the page more easier to read.
- In order to allow for more visual accessibility I changed the color contrast of the header background and header description such that the description is now text-gray-700 from the previous text-gray-600. I was wondering if this is okay @mapsandapps 

### Steps to test

- All tests are inside launchpad folder
1. Run yarn dev from the root folder (cypress)
2. Add the launchpad as a project on the Cypress UI
3. Go to e2e testing and run the project-setup.cy.ts spec file. This contains all updated e2e tests that check the default collapsible functionality.
4. Change to Component Testing and run the FileRow.cy.ts spec file. This contains all the component tests for FileRow where I made the changes.

### How has the user experience changed?

Before:
<img width="683" alt="Screen Shot 2022-09-01 at 11 40 57 AM" src="https://user-images.githubusercontent.com/65267668/187955788-f6f8e1c2-0be0-4f76-9ed4-faf0cabbb8aa.png">
After:
<img width="683" alt="Screen Shot 2022-09-01 at 11 39 26 AM" src="https://user-images.githubusercontent.com/65267668/187955417-5bfcd397-9909-492e-b081-9f0de0afc016.png">

### PR Tasks


- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
